recipe/meta.yaml

```diff
@@ -16,17 +16,23 @@ build:
     - vc9     # [win and py27]
     - vc10    # [win and py34]
     - vc14    # [win and py35]
+    - vc14    # [win and py36]
+  skip: true  # [win and py35]
 
 requirements:
   build:
+    - krb5 1.14.*
+    - libssh2 1.8.*
     # python is a build requirement on Windows to resolve features.
     - python           # [win]
     # perl is required to run the tests on UNIX.
-    - perl             # [unix]
+    - perl 5.22.2.1    # [unix]
     - pkg-config       # [unix]
     - openssl 1.0.*    # [unix]
     - zlib 1.2.*
   run:
+    - krb5 1.14.*
+    - libssh2 1.8.*
```

Comment:
Agreed `krb5` seems fine. I think `libssh2` also looks fine. They have added things in the past, but only when the minor number increased (excepting 1.2.x which is quite old). We don't have much (any) history to judge 1.8.x on, but am guessing they are trying to do semantic versioning.

The way that we finalize pinnings is merging them into that pinning script. After which we can make use of them elsewhere. As far as I'm concerned your recommendations seem good. Would just include this content in that PR including links to ABI Laboratory Tracker results. Let's see what others say and then we can finalize all of this.